### PR TITLE
If role ends in "| <sym>", all members' nicknames will end in "| <sym>"

### LINKED ISSUE #2: The Admin symbol
Whenever a user is assigned the admin role, the bot should append " ※" to their server nickname. Whenever their admin role is taken away, the symbol should be removed

### PR DESCRIPTION
**Pull request by abrahammurciano**
Closes #2

### CODE REVIEW

**Comment by abrahammurciano** [comment]
@DenverCoder1 to continue testing this I need Member Intents turned on for the bot.

**Comment by DenverCoder1** [comment]
Make sure to add any new config vars on Heroku as well

**Comment by abrahammurciano** [comment]
Sorry about the mess... i seem to have commits from other branches in here, not sure why...

File: modules/role_tag/member.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	def tags(self) -> List[str]:
+		roles = sorted(
+			self.inner_member.roles, key=lambda role: role.position, reverse=True
+		)
+		roles = [Role(role) for role in roles]
+		return [role.tag.strip() for role in roles if role.has_tag()]
+
+	def tags_str(self) -> str:
+		tags = self.tags()
+		return " " + self.sep + " " + " ".join(tags) if tags else ""
```
Comment (DenverCoder1) [inline modules/role_tag/member.py:29]: Can possibly simplify with f-string?

**Review by DenverCoder1 — APPROVED** [review]
Looks good

**Pull request merged**